src/main/interface/cli.c

```diff
@@ -2264,6 +2264,24 @@ static void cliName(char *cmdline)
     printName(DUMP_MASTER, pilotConfig());
 }
 
+static void printDisplayName(uint8_t dumpMask, const pilotConfig_t *pilotConfig)
+{
+    const bool equalsDefault = strlen(pilotConfig->displayName) == 0;
+    cliDumpPrintLinef(dumpMask, equalsDefault, "display name %s", equalsDefault ? emptyName : pilotConfig->displayName);
```

Comment:
This should output `display_name` not `display name`.  Otherwise the output is not usable to copy/paste back into the config.